plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/GetMobileApp.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks;
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Task;
+
+/**
+ * Get Mobile App Task
+ */
+class GetMobileApp extends Task {
+	/**
+	 * ID.
+	 *
+	 * @return string
+	 */
+	public function get_id() {
+		return 'get-mobile-app';
+	}
+
+	/**
+	 * Title.
+	 *
+	 * @return string
+	 */
+	public function get_title() {
+		return __( 'Get the free WooCommmerce mobile app', 'woocommerce' );
```

Comment:
```suggestion
		return __( 'Get the free WooCommerce mobile app', 'woocommerce' );
```

Tpyo!

Reply:
I reckon its the official unofficial spelling

![image](https://user-images.githubusercontent.com/27843274/189848587-95e11f31-0155-4288-a367-4cbcb388802a.png)

Reply:
(fixed)